.changeset/twenty-lamps-rule.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Maybe something like this ?
```suggestion
---
'@clerk/clerk-js': patch
---

Fix redirect flow for OAuth2 IDP flow with email_link verification
```

Reply:
Thanks @panteliselef, updated the changeset!